src/pypromice/resources/variables.csv

```diff
@@ -1,91 +1,91 @@
 field,standard_name,long_name,units,coverage_content_type,coordinates,instantaneous_hourly,where_to_find,lo,hi,OOL,station_type,L0,L2,L3,max_decimals
-time,time,Time,yyyy-mm-dd HH:MM:SS,physicalMeasurement,time lat lon alt,,,,,,all,1,1,1,
```

Comment:
Unfortunately, the stations' lat, lon, alt are changing with time, which makes unsuitable to be used as stable coordinates.
The CF convention is not very well designed for "floating" station like ours. I really don't want users to think the stations are static.

As a not for future enhancement, it would be great to have station_id or site_id as a coordinate (instead of a attribute), so that we could do things like
```python
ds = xr.open_mf(list_files, concat_dim='station_id')
ds.t_u.mean(dim='time') # gives for each station the average temperature
```